src/main/resources/pid.properties

```diff
@@ -63,12 +63,16 @@ database.sqlserver=pid-3569588c-b89d-5567-84ee-a2c633c7204c
 
 dynamic.aad.end=pid-f5a60b13-efd6-551a-a40f-3923804e475d
 dynamic.aad.start=pid-7dcc8904-9a8a-5b71-a73d-6caa9edb78ef
+dynamic.addnode.start=pid-5f68f633-701b-58b0-83c2-399c6d1d8414
+dynamic.addnote.end=pid-449e677a-5fc3-55cb-9193-4b225f42e646
 dynamic.cluster.addnode.end=pid-e9ac3048-2c62-5ae4-8a6d-fa6cc0d756ac
 dynamic.cluster.addnode.start=pid-9697621e-54df-51da-8169-ffcb58cb0a0c
 dynamic.cluster.end=pid-515d22ac-ff8c-5faa-aca1-3605ab5d2034
 dynamic.cluster.start=pid-22e98104-2229-5ec7-9a90-12edca3d88e7
 dynamic.database.end=pid-699d05c7-41f8-5512-aeef-f68651e247c2
 dynamic.database.start=pid-488f2f70-6999-5970-bae9-251097c2d0b6
+dynamic.deletenode.start=pid-931c2901-224a-5cc6-b235-41fa162b7532
+dynamic.deletenode.end=pid-3eb27f94-d1c3-572c-a7da-0d7f26f6a20e
```

Comment:
Hello @edburns we have to add those pids to oracle publisher in partner center, to track deleting nodes from dynamic cluster.

Reply:
@galiacheng done.